go.mod

```diff
@@ -183,7 +183,7 @@ require (
 replace (
 	github.com/facebookincubator/nvdtools => github.com/stackrox/nvdtools v0.0.0-20231111002313-57e262e4797e
 
-	github.com/heroku/docker-registry-client => github.com/stackrox/docker-registry-client v0.1.0
+	github.com/heroku/docker-registry-client => github.com/stackrox/docker-registry-client v0.2.1-0.20240809225852-a122013c5989
```

Comment:
https://github.com/stackrox/docker-registry-client/pull/21 should be merged and v0.2.1 should be cut prior to merging this PR